### DISCUSSION TITLE
Question about getting PowerShell return value

### DISCUSSION THREAD

**Discussion by ms000** [Tyrrrz/CliWrap#188] [Q&A]
First, thanks very much for creating CliWrap; it is fantastic!

I have been using it to build a library of PowerShell functions to automate some Azure tasks. Here is my question: I can't figure out how to get the return value of, for example this command:

 **$secretvalue = ConvertTo-SecureString "hVFkk965BuUv" -AsPlainText -Force**

I don't see "$secretvalue" in StandardOutput.

Any ideas would be appreciated.

Thanks,
Mark

`
      public static async Task<string> ConvertToSecureString(string inputString)
        {
            var result = await Cli.Wrap("powershell")
            .WithArguments(args => args
                .Add("ConvertTo-SecureString")
                .Add(inputString)
                .Add("-AsPlainText")  
                .Add("-Force")           
            )
            .ExecuteBufferedAsync();
            
            if (result.ExitCode == 0)
            {
                return result.StandardOutput;
            }
            else
            {
                return result.StandardError;
            }               
        }`

**Comment by Tyrrrz** [comment]
I'd imagine you need to print it with `Write-Host` or something similar.
